packages/ses/scripts/bundle.js

```diff
@@ -1,10 +1,14 @@
+/** @import {ModuleTransforms} from '../../compartment-mapper/types.js' */
```

Comment:
note: conflict with #2310; this would want to be `SyncModuleTransforms` instead.

also, if you're reaching in to `compartment-mapper` _from_ `ses` (which it appears creates a cyclic dependency), it seems like `compartment-mapper` (or somewhere else)  might want to be where this script lives.